app/views/nsm/imports/errors.html.erb

```diff
@@ -7,8 +7,9 @@
     <p><%= t('.explanation') %></p>
 
     <%# Ugly formatting because any whitespace in the block is passed through to the frontend %>
+    <% formatted_errors = JSON.parse(AppStoreClient.new.get_import_error(params[:error_id])['details']) %>
```

Comment:
This call should be in the controller, the view should recieve the data it needs

Reply:
yes you're right I don't know why I left it like this - I'll use an instance variable from the controller